### PR TITLE
5xx error handled and handle module permission exception handled

### PR DESCRIPTION
**Pull request by ruuushhh**

### CODE REVIEW

File: apps/mappings/tasks.py [hunk 1]
```diff
@@ -194,6 +194,8 @@ def auto_create_project_mappings(workspace_id: int):
     except FyleInvalidTokenError:
         logger.info('Invalid Token for fyle')
     
+    except InternalServerError:
+        logger.error('Internal server error fyle')
```
Comment (ashwin1111) [inline apps/mappings/tasks.py:198]: Internal server error while importing to Fyle

File: apps/mappings/tasks.py [hunk 2]
```diff
@@ -240,6 +242,9 @@ def async_auto_map_employees(workspace_id: int):
     except FyleInvalidTokenError:
         logger.info('Invalid Token for fyle')
 
+    except NoPrivilegeError:
+        logger.info('Insufficient permissions')
```
Comment (ashwin1111) [inline apps/mappings/tasks.py:246]: Insufficient permission to access the requested module

File: apps/sage_intacct/tasks.py [hunk 3]
```diff
@@ -643,6 +643,9 @@ def create_journal_entry(expense_group: ExpenseGroup, task_log_id):
     except InvalidTokenError as exception:
         handle_sage_intacct_errors(exception, expense_group, task_log, 'Journal Entry')
 
+    except NoPrivilegeError:
```
Comment (ashwin1111) [inline apps/sage_intacct/tasks.py:646]: club all exceptions that calls handle_sage_intacct_errors() in this file

File: apps/sage_intacct/tasks.py [hunk 4]
```diff
@@ -1271,6 +1297,10 @@ def check_sage_intacct_object_status(workspace_id):
         logger.info('Invalid Token or SageIntacct credentials does not exist - %s', workspace_id)
         return 
 
+    except NoPrivilegeError:
```
Comment (ashwin1111) [inline apps/sage_intacct/tasks.py:1300]: can club this to above exception, update message

File: apps/sage_intacct/tasks.py (hunk 3 above)
Comment (ruuushhh) [reply apps/sage_intacct/tasks.py:646]: Can you explain this a bit?

Comment (ashwin1111) [reply apps/sage_intacct/tasks.py:646]: wherever we're calling handle_sage_intacct_errors(), you can add this exception to that list. Example - 
```
except (InvalidTokenError, NoPrivilegeError) as exception:
        handle_sage_intacct_errors(exception, expense_group, task_log, 'Charge Card Transactions')

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**